src/main/kotlin/org/rust/cargo/project/CargoProjectDescription.kt

```diff
@@ -142,6 +142,13 @@ class CargoProjectDescription private constructor(
 
     fun findPackage(name: String): Package? = packages.find { it.name == name }
 
+    fun findLibTargetByNormName(normName: String): Pair<Package, Target>? =
+        packages.asSequence()
+            .mapNotNull { pkg ->
+                pkg.libTarget?.let { if (it.normName == normName) Pair(pkg, it) else null }
+            }
+            .firstOrNull()
```

Comment:
>How exactly do you think we can use this property? It can be both empty and filled for documented crates.

Really? I think it should be `"source": "registry+https://github.com/rust-lang/crates.io-index"` for all creates from crates.io?